src/extensions/default/AutoUpdate/UpdateInfoBar.js

```diff
@@ -113,6 +117,18 @@ define(function (require, exports, module) {
                 }
             }
         }
+        var resizeContentContainer = function resizeContentContainer() {
+            if($updateContent.length > 0) {
+                var newWidth = $updateBar.outerWidth() - $buttonContainer.outerWidth()
+                               - $iconContainer.outerWidth() - $closeIconContainer.outerWidth() - 38;
```

Comment:
What is this 38? Is this fixed number for every window size? All these values are in pixels right?